sdk/auth/interface.go

```diff
@@ -13,8 +13,8 @@ import (
 // Authorizer is anything that can return an access token for authorizing API connections
 type Authorizer interface {
 	Token(ctx context.Context, request *http.Request) (*oauth2.Token, error)
-
 	AuxiliaryTokens(ctx context.Context, request *http.Request) ([]*oauth2.Token, error)
+	ExpireTokens() error
```

Comment:
for this one (but also in retrospect) is it worth adding some documentation for this one? As it stands the "will do nothing" comments on each implementation will show as needed, but don't strictly explain the purpose for this one?

Reply:
good shout, will add 👍